src/entity/MarketingCta.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+export type MarketingCtaVariant = 'card' | 'popover';
+
+export type MarketingCtaFlags = Partial<{
```

Comment:
Do we maybe want to define which ones are required/not?

Reply:
Yeah, I've updated this type now